Loadouts/blu_usmc_m16_desert.hpp

```diff
@@ -0,0 +1,259 @@
+  };
+  class rhsusf_usmc_marpat_wd_gunner: Fic_Spotter {// Assistant Mortar
+    vest[] = { "rhsusf_spc_light" };
+    MORTAR_GEAR("B_Mortar_01_support_F")
+  };
+  class rhsusf_usmc_marpat_wd_helipilot {// Pilot
+    uniform[] = {"U_B_HeliPilotCoveralls"};
+    backpack[] = {"B_AssaultPack_blk"};
+    vest[] = {"V_TacVest_blk"};
+    headgear[] = {"rhsusf_hgu56p"};
+    weapons[] = {WEST_SMG};
+    magazines[] = {WEST_SMG_MAG,CREW_GRENADES};
+    backpackItems[] = {KEY_WEST,RADIO_LR};
+    items[] = {BASE_MEDICAL,WEST_BASE_TOOLS,WEST_LEADER_TOOLS,RADIO_MR};
+    linkedItems[] = {WEST_BASE_LINKED,WEST_LEADER_LINKED};
+  };
+  class rhsusf_usmc_marpat_wd_helicrew: rhsusf_usmc_marpat_wd_helipilot {// Pilot Crewchief
+    headgear[] = {"rhsusf_hgu56p_mask"};
+  };
+  class rhsusf_usmc_marpat_wd_combatcrewman: Fic_Soldier_Carbine {// Crewman Commander
+    vest[] = {"rhsusf_spc_crewman"};
+    headgear[] = {"rhsusf_cvc_helmet"};
+    backpack[] = {"B_Carryall_mcamo"};
+    magazines[] = {WEST_CARBINE_MAG,CREW_GRENADES};
+    backpackItems[] = {KEY_WEST,RADIO_LR};
+    linkedItems[] = {WEST_BASE_LINKED,WEST_LEADER_LINKED,BINOS};
+    items[] += {BASE_MEDICAL};
+  };
+  class rhsusf_usmc_marpat_wd_crewman: rhsusf_usmc_marpat_wd_combatcrewman {// Crewman Gunner
+    backpackItems[] = {RADIO_MR,KEY_WEST};
+    linkedItems[] = {WEST_BASE_LINKED,WEST_LEADER_LINKED};
+  };
+  class rhsusf_usmc_marpat_wd_engineer: rhsusf_usmc_marpat_wd_combatcrewman{// Crewman Driver
+    backpackItems[] += {"Toolkit"};
+  };
+  class Fic_eng: rhsusf_usmc_marpat_wd_crewman {
+    headgear[] = {"rhsusf_lwh_helmet_marpatd"};
+    items[] += {BASE_ENG};
+    backpackItems[] = {};
+  };
+  class rhsusf_usmc_marpat_wd_explosives: Fic_eng{// EOD Explodey guy
+    vest[] = { "rhsusf_spc_rifleman" };
+    //backpackItems[] += {BASE_EXP};    // uncomment to add explosives
+    //backpackItems[] += {BASE_MINE};   // uncomment to add mines
+    linkedItems[] = { WEST_BASE_LINKED };
+  };
+
+  class rhsusf_usmc_recon_marpat_wd_teamleader: rhsusf_usmc_marpat_wd_squadleader {// SF Squad Leader
+    //headgear[] = { "rhsusf_Bowman" };
+  };
+  class rhsusf_usmc_recon_marpat_wd_teamleader_fast: rhsusf_usmc_marpat_wd_teamleader{// SF FTL
+    //headgear[] = {"rhsusf_Bowman"};
+  };
+  class rhsusf_usmc_recon_marpat_wd_machinegunner: rhsusf_usmc_marpat_wd_autorifleman_m249{// SF AR
+  };
+  class rhsusf_usmc_recon_marpat_wd_marksman: rhsusf_usmc_marpat_wd_autorifleman_m249_ass{// SF AAR
+    weapons[] = {WEST_SNIPER};
+    magazines[] = {WEST_SNIPER_MAG, WEST_BASE_GRENADES};
+    attachments[] = {};
+  };
+  class rhsusf_usmc_recon_marpat_wd_rifleman_at: rhsusf_usmc_marpat_wd_riflemanat{// SF RAT
+  };
+
+
+  class fallback: rhsusf_usmc_marpat_wd_rifleman {}; //This means any faction member who doesn't match something will use this loadout
+};
+
+class rhs_faction_usn : rhs_faction_usmc_wd{
+  class rhsusf_navy_marpat_wd_medic : Fic_Soldier_Carbine{// SL Corpsman
```

Comment:
If you mean does the loadout properly apply with the medical gear, then yes, it works.